webextensions/api/scripting.json

```diff
@@ -3,7 +3,6 @@
     "api": {
       "scripting": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/",
```

Comment:
@Elchi3 this link is valid, no redirect that I can see

Reply:
Yeah, there was a trailing slash, however.